cmd/minishift/cmd/start.go

```diff
@@ -787,25 +774,17 @@ func populateStartFlagsToViperConfig() {
 // determineClusterUpParameters returns a map of flag names and values for the cluster up call.
 func determineClusterUpParameters(config *clusterup.ClusterUpConfig, DockerbridgeSubnet string) map[string]string {
 	clusterUpParams := make(map[string]string)
-	valid, _ := openshiftVersion.IsGreaterOrEqualToBaseVersion(config.OpenShiftVersion, constants.RefactoredOcVersion)
-	if valid {
-		// Set default value for base config for 3.10
-		clusterUpParams["base-dir"] = baseDirectory
-		if viper.GetString(configCmd.ImageName.Name) == "" {
-			imagetag := fmt.Sprintf("'%s:%s'", minishiftConstants.ImageNameForClusterUpImageFlag, config.OpenShiftVersion)
-			viper.Set(configCmd.ImageName.Name, imagetag)
-		}
-		// Add docker bridge subnet to no-proxy before passing to oc cluster up
-		// This will be removed once proxy issue fixed on upstream side for oc cluster up.
-		// https://github.com/openshift/origin/issues/20496
-		if viper.GetString(configCmd.NoProxyList.Name) != "" {
-			viper.Set(configCmd.NoProxyList.Name, fmt.Sprintf("%s,%s", DockerbridgeSubnet, viper.GetString(configCmd.NoProxyList.Name)))
-		}
-	} else {
-		// This will only required to work with openshift < 3.10
-		for key, value := range dataDirectory {
-			clusterUpParams[key] = value
-		}
+	// Set default value for base config for 3.10
+	clusterUpParams["base-dir"] = baseDirectory
+	if viper.GetString(configCmd.ImageName.Name) == "" {
+		imagetag := fmt.Sprintf("'%s:%s'", minishiftConstants.ImageNameForClusterUpImageFlag, config.OpenShiftVersion)
+		viper.Set(configCmd.ImageName.Name, imagetag)
+	}
+	// Add docker bridge subnet to no-proxy before passing to oc cluster up
+	// This will be removed once proxy issue fixed on upstream side for oc cluster up.
+	// https://github.com/openshift/origin/issues/20496
```

Comment:
@praveenkumar I do not think this will be ever fixed from oc cluster up side.

Reply:
So I guess we can remove the reference.

Reply:
@LalatenduMohanty sure, you can remove this comment during merge.